rpcs3/Emu/Cell/lv2/sys_net/lv2_socket_p2ps.h

```diff
@@ -87,7 +90,10 @@ class lv2_socket_p2ps final : public lv2_socket_p2p
 	s32 poll(sys_net_pollfd& sn_pfd, pollfd& native_pfd) override;
 	std::tuple<bool, bool, bool> select(bs_t<poll_t> selected, pollfd& native_pfd) override;
 
-protected:
+private:
+	void close_stream_nl(nt_p2p_port* p2p_port);
+
+private:
```

Comment:
?

Reply:
Separate functions and variables, you know I do this :D